### PR TITLE
Simplify the closing of modals.

### PR DESCRIPTION
**Pull request by realityking**
Using Angular 1.3 and current master of UI Bootstrap closing modals did not work. Easily fixed using `$dismiss`.

Nice side effect, the code is a bit shorter and simpler.

Since this is my first pull request, do you want the dist folder updated or rather not?

### CODE REVIEW

File: src/scripts/dashboard.js [hunk 1]
```diff
@@ -306,11 +305,10 @@ angular.module('adf')
             instance.close();
```
Comment (sdorra) [inline src/scripts/dashboard.js]: Do we need to call $dismiss here as well?

**Comment by sdorra** [comment]
At the moment i update dist only on releases.

**Comment by blkeagl** [comment]
This is a bug that appears closing all modals in the framework when using angular 1.3.x (I am using 1.3.15) and the newly released ui-bootstrap 0.13.0. I was about to create a pull request to fix this myself, but will happily let this be fixed here.

**Comment by sdorra** [comment]
I can't reproduce the issue. I've tried with angular 1.3.15 and ui-bootstrap 0.13.0.

**Comment by blkeagl** [comment]
I reproduced it and logged a bug with the ui-bootstrap team (https://github.com/angular-ui/bootstrap/issues/3624) which they closed saying that the $new implementation is incorrect implementation and why the dialogs won't close. I'm surprised you cannot reproduce. I can try taking it again.

**Comment by sdorra** [comment]
I've updated angular and ui-bootstrap of the adf master branch and adf-dynamic-example, but without problems. I've read [angular-ui/bootstrap#3624](https://github.com/angular-ui/bootstrap/issues/3624). But we could not rely on the scope which is created by the modal itself, because we have to add properties to the modal scope, which are not available in the parent scope. Could some post an non working example?

**Comment by adamdva** [comment]
We are using ui-bootstrap 0.13.0 and using $modal also. Just tried upgrading to angular 1.4.0 and now see this issue. The modal appears to close, but the element is still there. We are using backdrop static, so the page is then unresponsive, and the divs are still found when using the developer element explorer.

**Pull request closed without merging**